src/containers/RecentTrades/Market.tsx

```diff
@@ -49,17 +50,21 @@ const handleHighlightValue = (prevValue: string, curValue: string) => {
 };
 
 
-class MarketComponent extends React.Component<Props> {
+class RecentTradesMarketContainer extends React.Component<Props> {
+    public componentDidMount() {
+        if (this.props.currentMarket) {
+            this.props.tradesFetch(this.props.currentMarket);
+        }
+    }
```

Comment:
maybe we should not fetch info in didMount to prevent asynchronous actions. Willl be enough to fetch trade in componentWillReceive props ?

Reply:
right now it`s ok